### DISCUSSION TITLE
Do we have an output to the magnitude of the QTL effects?

### DISCUSSION THREAD

**Discussion by alanaselli** [gaynorr/AlphaSimR#145] [Q&A] [answered]
Hello everyone,

I would like to know if the QTLs generated by ```SP$addTraitA(10)``` have all the same effect on the trait, or if the effects follow a normal distribution or any other rule. In other words, if I simulate an additive trait with 10 QTLs, will each be responsible for exactly 10% of the additive variance?

I was able to output the QTL locations, however I couldn't find any information on their effects. This would be important to contrast with GWAS results while studying about this topic. Is it possible to obtain this information somehow?

Additionally, would it be possible to control the size of the effects of each QTL directly (as opposed to setting mean and variance for the trait, which I believe affect the effect sizes of QTLs as a whole)? I have considered using ```SimParam$importTrait``` to set each QTL effect individually, although I haven't attempted this yet. Any suggestions are appreciated.

Thank you very much for your time!
Alana

**Comment by gregorgorjanc** [comment, marked as answer]
@alanaselli see this 

```
> #Create founder haplotypes
> founderPop = quickHaplo(nInd=10, nChr=1, segSites=10)
> #Set simulation parameters
> SP = SimParam$new(founderPop)
> SP$addTraitA(10)
> SP$traits
[[1]]
An object of class "TraitA"
Slot "addEff":
 [1] -0.19952892 -0.35820658  0.70717565  0.34210157
 [5]  0.28970031  0.08040327  0.62986622  0.27986490
 [9]  0.68571113  0.84299564

Slot "intercept":
[1] 0.1159337

Slot "nLoci":
[1] 10

Slot "lociPerChr":
[1] 10

Slot "lociLoc":
 [1]  1  2  3  4  5  6  7  8  9 10

Slot "name":
[1] "Trait1"
```

You might want to consider doing the AlphaSimR intro course;) https://www.edx.org/course/breeding-programme-modelling-with-alphasimr

To set the effects, you would have to go via the import option indeed.

**Reply by alanaselli** [reply]
Thank you, @gregorgorjanc!
